package/Codegen/EventArgsTypeProperties.cs

```diff
@@ -74,40 +74,89 @@ struct EventArgsProperty {
             
             #line 30 "C:\Users\asklar\source\repos\react-native-xaml\package\Codegen\EventArgsTypeProperties.tt"
  foreach (var prop in Properties) { 
+if (prop.Property != null) { 
             
             #line default
             #line hidden
             this.Write("    { \"");
             
-            #line 31 "C:\Users\asklar\source\repos\react-native-xaml\package\Codegen\EventArgsTypeProperties.tt"
-            this.Write(this.ToStringHelper.ToStringWithCulture(Util.ToJsName(prop.GetName())));
+            #line 32 "C:\Users\asklar\source\repos\react-native-xaml\package\Codegen\EventArgsTypeProperties.tt"

```

Comment:
I'm confused why we have all these hard coded paths in here, is this file meant to be checked in?

Reply:
yes this is an autogenerated file (generated from the corresponding .tt file)